recipes/pybinding/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "pybinding" %}
+{% set version = "0.8.1.dev0" %}
```

Comment:
I tried initially with a non-dev release, but it didn't build correctly. If a non-dev release and tarball are preferable, I can close this PR now and wait for the next release version.